__tests__/index.test.js

```diff
@@ -75,6 +75,25 @@ describe("GET /api/articles/:article_id", () => {
         expect(body.msg).toBe("Bad Request");
       });
   });
+
+  test("GET 200: returns article with the article ID passed with an added key of comment_count", () => {
```

Comment:
I can't see your 404 and 400 tests. Please add these if you haven't written them